tests/app/submitter/test_submitter.py

```diff
@@ -212,6 +212,28 @@ def test_gcs_submitter_adds_metadata_when_sends_message(patch_gcs_client):
     }
 
 
+def test_gcs_submitter_adds_receipting_keys_to_metadata_when_set(patch_gcs_client):
```

Comment:
This no longer has the concept of receipting keys. It is just that it sets additional metadata.